vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -82,3 +82,9 @@ struct Balance {
   map<string, double> rates;
   map<string, double> wallets;
 };
+
+struct MediaEventInfo {
+  string event_;
```

Comment:
May want to consider taking this refactor time to remove trailing underscore's? I don't see them used in any other mojo struct yet.

Reply:
sgtm, will remove